docs/customizing_attribute_partials.md

```diff
@@ -54,3 +54,17 @@ Changing numbers to display to three decimal places might look like this:
 
 If you only want to change how an attribute appears
 on a single page (e.g. `index`), you may delete the unnecessary templates.
+
+## Using a Custom Look
+
+If you want to generate a different look, run the following:
+
+```bash
+rails generate administrate:views:field number --look custom
```

Comment:
Does this actually work? I've been trying today and it didn't work, and only now I'm realising perhaps this generator isn't supposed to run on its own.